src/ContextualPopup/ContextualPopup.js

```diff
@@ -568,10 +606,10 @@ module.exports = kind(
 		// show a transparent scrim for modal popups if
 		// {@link module:moonstone/ContextualPopup~ContextualPopup#scrimWhenModal} is `true`, else show a
 		// regular scrim.
-		if (this.modal && this.scrimWhenModal) {
+		if (this.modal && this.scrimWhenModal && !this.scrim) {
```

Comment:
Yes, and I think that is caused because current scrim is creating separate instance for transparent and translucent. If we don't have a plan to have single instance for both, then I think we need to hide previous scrim before showing other kind of scrim. 

No problem in below case:
1) Popup A is showing transparent scrim then hide.
2) Popup B is showing translucent scrim then hide.

Problem in below case.
1) Popup A is showing transparent scrim.
2) Popup A is change this.scrim property to show translucent scrim (new instance).